internal/docker/metadata.yaml

```
@@ -1,3 +1,4 @@
 status:
   codeowners:
-    active: [rmfitzpatrick, jamesmoessis]
\ No newline at end of file
+    active: [jamesmoessis]
```

Feel free to add me to this:

```suggestion
    active: [jamesmoessis, moviestoreguy]
```